### PR TITLE
Rk 295 utils insert db

### PR DESCRIPTION
**Pull request by Robert-Krajcik**

### CODE REVIEW

**Review by Jeff-Thompson12 — COMMENTED** [review]
Looks pretty good. I think my only "complaint" would be that the tests don't check that the `tryCatch()` for the appropriate functions. Theoretically, I'm not sure if it should ever be triggered though.

File: tests/testthat/test-utils_insert_db.R [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  
+  # valid db? test this is a valid sqlite database
+  testthat::expect_equal(readLines(file.path(base_path, db_temp), n =1, warn = FALSE), "SQLite format 3")
+  
+  # load pkg info for stringr into the database
+  pkg_name <- "stringr"
+  
+  insert_pkg_info_to_db(pkg_name, file.path(base_path, db_temp))
+
+  test_that("get_pkg_info works", {
```
[the remark this answers is not in the document]
Comment (Robert-Krajcik) [reply tests/testthat/test-utils_insert_db.R]: will update the comment

File: tests/testthat/test-utils_insert_db.R [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                    WHERE package_metrics.package_id = '{pkg_id}' AND 
+                    metric.class = 'maintenance' ;"), file.path(base_path, db_temp))
+    expect_s3_class(mmdata, "data.frame")
+    expect_equal(names(mmdata), c("name", "long_name", "description", "is_perc", "is_url", "value"))
+    expect_equal(mmdata$name[1], "has_vignettes")
+  })
+
+  insert_community_metrics_to_db(pkg_name, file.path(base_path, db_temp))
+  
+  test_that("get_comm_data works", {
```
[the remark this answers is not in the document]
Comment (Robert-Krajcik) [reply tests/testthat/test-utils_insert_db.R]: will update the comment

**Review by Jeff-Thompson12 — APPROVED** [review]
LGTM!

**Pull request merged**